plugins/utils/agol_utils.py

```diff
@@ -1,5 +1,36 @@
 import requests
-
+import logging
```

Comment:
I made changes to this utils file to better deal with geometry fields from AGOL, based on a snippet you wrote - I think I'll put it all into the AGOLOperator one day to make it all cleaner

Reply:
Agreed, good thinking. My end goal once I discovered custom operators was to remove utils and make it an operator as well.

It makes sense because the only time this code runs is when we get data from AGOL, which would be a task as a part of a DAG. At the moment I can't think of why the function would be used standalone otherwise.

That said, this logic should be the base of that AGOL operator (which I believe is what we already have).

Could you please remind me why we need to keep this function, again? Could we delete it in favour of the operator?

Reply:
yeah you know what Im overcomplicating this

I've removed the dependence on the `utils.agol_utils` module, taken those old methods and placed them into the class itself.

Thank you for the push to do this - I should haev done it sooner.